Tests/Card Tests/CardComponentTests.swift

```diff
@@ -2140,14 +2140,15 @@ class CardComponentTests: XCTestCase {
     private var shopperInformation: PrefilledShopperInformation {
         let billingAddress = PostalAddressMocks.newYorkPostalAddress
         let deliveryAddress = PostalAddressMocks.losAngelesPostalAddress
-        let shopperInformation = PrefilledShopperInformation(shopperName: ShopperName(firstName: "Katrina", lastName: "Del Mar"),
-                                                             emailAddress: "katrina@mail.com",
-                                                             telephoneNumber: "1234567890",
-                                                             billingAddress: billingAddress,
-                                                             deliveryAddress: deliveryAddress,
-                                                             socialSecurityNumber: "78542134370",
-                                                             card: .init(holderName: "Katrina del Mar"))
-        return shopperInformation
+        return .init(
+            shopperName: ShopperName(firstName: "Katrina", lastName: "Del Mar"),
```

Comment:
Minor suggestion:
We can move this `PrefilledShopperInformation` instance to `Dummy` file and reuse it instead of copy-pasting it all over the tests.